spec/assessments/calculateFleschReadingSpec.js

```diff
@@ -0,0 +1,17 @@
+var fleschReadingAssessment = require( "../../js/assessments/calculateFleschReading.js" );
+var Paper = require( "../../js/values/Paper.js" );
+
+var factory = require( "../helpers/factory.js" );
+var i18n = factory.buildJed();
+
+describe( "An assessment for the fleschReading", function(){
```

Comment:
Please consider adding tests for cases with a reading ease score like: 0, 100 and 60 (the line between two scores)